Networking/Networking/Mapper/ShippingLabelPrintDataMapper.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+
+/// Mapper: Shipping Label Print Data
+///
+struct ShippingLabelPrintDataMapper: Mapper {
+    /// (Attempts) to convert a dictionary into ShippingLabelPrintData.
```

Comment:
nit:
```suggestion
    /// (Attempts) to convert a dictionary into ShippingLabelPrintDataMapper.
```

Reply:
I think we usually call it the entity/model name that the mapper is converting to, like:

https://github.com/woocommerce/woocommerce-ios/blob/5c93e7b56fbc9bd513f0971e1b3e55e4d1b32a94/Networking/Networking/Mapper/OrderCountMapper.swift#L14